tensorflow_datasets/image/nyu_depth_v2_test.py

```diff
@@ -0,0 +1,17 @@
+"""Tests for NYU Depth V2 Dataset."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import nyu_depth_v2
+
+
+class NyuDepthV2Test(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = nyu_depth_v2.NyuDepthV2
+  SPLITS = {"train": 2, "test": 1}
```

Comment:
Tests Failing !!!!!!
Suggested Changes - please change to - 
`SPLITS = {"train": 2, "validation": 1}`

Reply:
Thanks, fixed in 46835075ad7dd6dd4b0478877f57b07e0e282add.